src/screens/Assets/liquidityPools/selectors.js

```diff
@@ -41,18 +45,7 @@ export function useLiquidityPoolsChainBalances(): TotalBalancesPerChain {
   return useRootSelector(liquidityPoolsTotalBalanceByChainsSelector);
 }
 
-export type LiquidityPoolItem = {|
-  key: string,
-  service: string,
-  title: string,
-  iconUrl: string,
-  value: BigNumber,
-  change?: BigNumber,
-|};
-
-export function useLiquidityPoolAssets(): ChainRecord<LiquidityPoolItem[]> {
-  // TODO: replace once available from Etherspot SDK
-  const liquidityPools = [];
-
-  return { ethereum: liquidityPools };
+export function useLiquidityPoolAssets(): ChainRecord<LiquidityPoolAssetBalance[]> {
+  const assetsBalances = useRootSelector(accountAssetsBalancesSelector);
+  return getChainAssetsBalancesForCategory(assetsBalances, ASSET_CATEGORY.LIQUIDITY_POOLS);
```

Comment:
```suggestion
  return getChainAssetsBalancesForCategory(useRootSelector(accountAssetsBalancesSelector), ASSET_CATEGORY.LIQUIDITY_POOLS);
```
Maybe? Feels like a lot going on for one line, however...

Reply:
👍